test/README.md

```diff
@@ -57,7 +57,9 @@ The tests also expect to find several _secret_ parameters. While these can techn
 |------|----------|-------------|
 | `[E2E_]DPS_GROUP_KEY` | * | The symmetric key of the DPS [enrollment group](https://docs.microsoft.com/en-us/azure/iot-dps/concepts-service#enrollment-group) to use. Required when running any DPS tests, ignored otherwise. |
 | `[E2E_]IOT_HUB_CONNECTION_STRING` | ✔ | Hub-scoped IoT Hub connection string that can be used to get/add/remove devices, deploy edge configurations, and get/update module twins. |
+| `[E2E_]PREVIEW_IOT_HUB_CONNECTION_STRING` | * | Hub-scoped IoT Hub connection string in the Central EUAP region for preview testing - required only for PlugAndPlay tests. |
```

Comment:
This variable doesn't have to point to a hub in Central EUAP; that's just how you're using it in practice. The requirement is that you specify a hub that has PnP preview enabled on it.